coral/src/app/features/approvals/topics/TopicApprovals.tsx

```diff
@@ -78,15 +94,34 @@ function TopicApprovals() {
       />
     ) : undefined;
 
+  const selectedTopicRequest = topicRequests?.entries.find(
+    (request) => request.topicid === Number(detailsModal.topicId)
+  );
   return (
-    <ApprovalsLayout
-      filters={filters}
-      table={table}
-      pagination={pagination}
-      isLoading={topicRequestsLoading}
-      isErrorLoading={topicRequestsIsError}
-      errorMessage={topicRequestsError}
-    />
+    <>
```

Comment:
I would add a test case here, because now `TopicsApproval` is responsible for showing a details modal with the right data. 

For Schema Request I did it like that: 
https://github.com/aiven/klaw/blob/3aa8f781b6d5d52d0cb4a48f39ebe1364dcde0be/coral/src/app/features/approvals/schemas/SchemaApprovals.test.tsx#L328

Reply:
That should work perfectly ^^ Thank you!

Reply:
[1c5d874](https://github.com/aiven/klaw/pull/669/commits/1c5d874726eb0590ec004e56cf3145b0ad3dda47)